### ISSUE TITLE
Add a procedure to migrate Hive tables to Iceberg

### ISSUE THREAD

**Issue by kiersten-stokes** [prestodb/presto#21632]
## Expected Behavior or Use Case
The `migrate` procedure would add Iceberg metadata to an existing Hive table, preserving the properties of the Hive table without duplicating the data

**Comment by agrawalreetika** [comment]
Hi @kiersten-stokes, 
I think there is a mismatch in Issue Title and description?
If my understanding is correct then `migrate` procedure is for migrating Hive table to Iceberg table?

**Comment by kiersten-stokes** [comment]
@agrawalreetika yes, that is my understanding as well!  I believe my description was correct but slightly confusing, so I've edited it to hopefully be more clear :)
